check.go

```diff
@@ -199,7 +203,27 @@ func (check *Check) web(name *string, sleep *time.Duration) {
 
 // The actual HTTP GET.
 func (check *Check) fetch() error {
-	resp, err := http.Get(check.Web)
+	var resp *http.Response
+	var err error
+	method := "GET"
+	if len(check.Headers) > 0 {
+		client := http.Client{}
+		var body io.Reader
+		if len(check.Body) > 0 {
```

Comment:
I bet, this means that `body` won't work without `headers`. If that's intended (I believe, not), what's the reason?